### ISSUE TITLE
Add unspecified: bottomAlphabetical option to rule-properties-order

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#778]
Would allow users who prefer alphabetical ordering to bump a couple of "special" properties to the top e.g.

```json
{
  "rule-properties-order": [
     [
       "all",
       "compose"
     ], {
        "unspecified": "bottomAlphabetical"
     }
  ]
}
```

```css
.a {
  all: initial;
  compose: .b;
  align-items: flex-end;
  display: block;
}
```

As well as letting those users who prefer a specific order to add strict alphabetical order to anything they've forgotten.

I believe it could be done with [a single extra conditional](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-properties-order/index.js#L156) (plus tests).

**Comment by davidtheclark** [comment]
Sounds good.
